File: Flutter_Stock_Original/lib/models/supplier.dart

class Supplier {
  String f_supID;
  String f_supName;
  String f_supDetail;
  int f_supStatus;

  Supplier(
      {required this.f_supID,
      required this.f_supName,
      required this.f_supDetail,
      required this.f_supStatus});

  factory Supplier.fromJson(Map<String, dynamic> json) {
    return Supplier(
        f_supID: json['f_supID'],
        f_supName: json['f_supName'],
        f_supDetail: json['f_supDetail'],
        f_supStatus: json['f_supStatus']);
  }
}

File: online_node_api/models/supplier.js
'use strict';
const {
  Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
  class Supplier extends Model {
    /**
     * Helper method for defining associations.
     * This method is not a part of Sequelize lifecycle.
     * The `models/index` file will call this method automatically.
     */
    static associate(models) {
      // define association here
    }
  };
  Supplier.init({
    f_supID: {
      type: DataTypes.STRING(200),
      primaryKey:true,
      allowNull:false,
      unique:true
    },
    f_supName: DataTypes.STRING,
    f_supDetail: DataTypes.STRING,
    f_supStatus: DataTypes.INTEGER
  }, {
    sequelize,
    modelName: 'Supplier',
    tableName: 'sys_supplier',
    timestamps: false
  });
  return Supplier;
};